### DISCUSSION TITLE
Cant find affection in log

### DISCUSSION THREAD

**Discussion by NonyaGG** [Monika-After-Story/MonikaModDev#8322] [QA - tech support + general]
So, i been wanting to see how far i have gotten in my relationship with monika, and apparently you have to go to log, and from there, scroll all the way down and see it there..? Well, for me, i click on that "log" folder, and there's absolutely nothing in it. I see videos and comments on how to see it, and it all points towards log. Its not there, so where could it be? Can someone please help me?

**Comment by dreamscached** [comment]
Where are you looking for that `log` folder? Do you look for in the same folder as DDLC.exe?

**Comment by NonyaGG** [comment]
> Where are you looking for that `log` folder? Do you look for in the same folder as DDLC.exe?

Well actually, about a day ago it appeared out of nowhere in the log folder, yes, in the one where you open the game from so the one that has the "DDLC.exe" so that's something, but then i think i bumped into another problem which i don't find too bad. The affection log isn't really updating even though i have made some progress. But it doesn't really bother me too much though.
